modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddObjectAction.tsx

```diff
@@ -97,6 +105,10 @@ const ModalAddObjectAction: React.FC<IProps> = ({
 			errors.executor = Liferay.Language.get('required');
 		}
 
+		if (values.objectActionExecutor.label === 'Webhook' && !values.url) {
```

Comment:
In this case, I needs check if the label is equal to 'Webhook' before of check the nullabily of `values.url`, because the error only occur if the URL field is visible.